adp/protocol/adp.py

```
@@ -0,0 +1,416 @@
+            return VerificationReport(
+                result=VerificationResult.INVALID,
+                entity_id=entity_id,
+                challenge_id=None,
+                checks_performed=['registration'],
+                checks_passed=[],
+                checks_failed=['registration'],
+                reason="Entity not registered",
+                timestamp=time.time()
+            )
+            
+        entity = self._entities[entity_id]
```

Ruff (0.13.1)

Local variable `entity` is assigned to but never used

Remove assignment to unused variable `entity`

(F841)